### PR TITLE
Fix hover styling on BCD signaling section

### LINKED ISSUE #6064: Hover styling should affect icon border width [BCD signalling]
On live, when a browser icon is hovered over or has focus, the border changes to MDN blue.

Screenshot 2019-10-17 at 10 24 47

Signifying focus with just a colour change isn't sufficient for accessibility so may we have the border change thickness similar to how it is on Zeplin:

Screenshot 2019-10-17 at 10 27 31

In the design the hover state changes the border form default `2px` width to `3px`, I'm anticipating that making this animate/transition may look a bit odd, so I'm happy to have no transition between these hover styles. Also happy to see `4px` if that works better!

### PR DESCRIPTION
**Pull request by mindy**
- Apply a thicker border width when hovering over browser icon

**Before:** 
hover-before

**After:** 
hover-after

Closes #6064

### CODE REVIEW

**Comment by schalkneethling** [comment]
@mindy I notice that the `kumascript` and `locale` submodule updates sneaked into the PR. Because this is such a small change, it is probably easiest to just do the following:

```
git checkout master
git branch -D bcd-border-6064
git pull origin master
git submodule update --init --recursive
git checkout -b bcd-border-6064

-- apply your change

git add .
git commit -m 'Message'
git push origin bcd-border-6064 (might/probably need a force push)
```

**Comment by mindy** [comment]
Thanks @schalkneethling ! I followed your instructions and pushed up a new version without the submodules.

File: kuma/static/styles/components/compat-tables/_bc-signal.scss [hunk 1]
```diff
@@ -463,9 +463,11 @@ div.bc-signal-block {
             transition: border-color .1s;
         }
 
-        &:hover {
+        &:hover:not(.selected) {
             .browser-logo {
                 border-color: #3d7e9a;
```
Comment (schalkneethling) [inline kuma/static/styles/components/compat-tables/_bc-signal.scss]: While you are working on this file, mind changing this to

```suggestion
                border-color: $blue;
```

Comment (mindy) [reply kuma/static/styles/components/compat-tables/_bc-signal.scss]: Thanks for the suggestion @schalkneethling! Did a search within the file and replaced all instances (only 4) with `$blue`.

**Pull request merged**